### PR TITLE
Make monitoring plugins Python3 compatible

### PR DESCRIPTION
**Pull request by lamaral**
igcollect is using this plugin on the monitoring servers. 
As we deprecate Python2, I made them compatible with Python3 also.

### CODE REVIEW

File: igcollect/artfiles.py [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-Copyright (c) 2016 InnoGames GmbH
+Copyright (c) 2019 InnoGames GmbH
 """
 
+import base64
 from argparse import ArgumentParser
 from time import time
-from urllib import urlopen
+
+try:
```
Comment (TheFRedFox) [inline igcollect/artfiles.py]: Please add a comment which of these modules are for Python2 and which are for Python3.

File: igcollect/artfiles.py [hunk 2]
```diff
@@ -1,12 +1,18 @@
-#!/usr/bin/env python
```
Comment (TheFRedFox) [inline igcollect/artfiles.py]: Why did you remove this line? PEP8 says keep python if it is python2 and python3.

File: igcollect/artfiles.py [hunk 3]
```diff
@@ -31,21 +37,23 @@ def parse_args():
...
     args = parse_args()
-    artfiles_url = (
-        'https://{}:{}@dcp.c.artfiles.de/api/stats/get_estats.html'
-        .format(args.http_user, args.http_pass)
-    )
-    data = urlopen(artfiles_url)
+    request = Request('https://dcp.c.artfiles.de/api/stats/get_estats.html')
+    credentials = '{}:{}'.format(args.http_user, args.http_pass)
+    base64credentials = base64.b64encode(credentials.encode())
+    request.add_header('Authorization',
```
Comment (TheFRedFox) [inline igcollect/artfiles.py]: Why aren't you using the `auth` parameter of the constructor of Request?
https://github.com/kennethreitz/requests/blob/master/requests/models.py#L214

Comment (TheFRedFox) [reply igcollect/artfiles.py]: Ah, nevermind, you are using the urllib module not the request module. But then, one could consider to use the request module. ^^

File: igcollect/artfiles.py [hunk 4]
```diff
@@ -31,21 +37,23 @@ def parse_args():
...
+                       'Basic {}'.format(base64credentials.decode()))
+    data = urlopen(request)
     template = args.prefix + '.{}.{}.{}.{} {} ' + str(int(time()))
```
Comment (TheFRedFox) [inline igcollect/artfiles.py]: Would prefer to have named template entries, though not in PR scope. So, feel free to do it, but not a must have.

File: igcollect/artfiles.py (hunk 3 above)
Comment (lamaral) [reply igcollect/artfiles.py]: I used urllib to avoid having external dependencies.

Comment (TheFRedFox) [reply igcollect/artfiles.py]: Quite sure, that we kinda habe the request module in every place, as it is so common. But you are right, that we do not need it. Keep it like that. Good to go like that.

File: igcollect/artfiles.py (hunk 2 above)
Comment (lamaral) [reply igcollect/artfiles.py]: I removed it because the script is executed by explicitly calling the Python binary. `/usr/bin/python3 artfiles.py`.

But looking at PEP8, it should be there in the main file, so I'll add it back.

File: igcollect/artfiles.py (hunk 4 above)
Comment (lamaral) [reply igcollect/artfiles.py]: Fixed 😃

File: igcollect/fastly.py [hunk 5]
```diff
@@ -114,7 +115,7 @@ def main(args):
         regions,
         grequests.map((get_service_data_request(api_key, service, {
             'from': start_time, 'to': end_time,
-            'by': interval, 'region': region,
+            'by':   interval, 'region': region,
```
[the remark this answers is not in the document]
Comment (lamaral) [reply igcollect/fastly.py]: Editor's fault. It's now gone.
